### ISSUE TITLE
Does not work with CRLF-terminated lines

### ISSUE THREAD

**Issue by mardukbp** [xinz/sax_map#3]
```elixir
> xml = "<mediawiki xml:lang=\"en\">\r\n  <page>\r\n    <title>Page title</title>\r\n    <revision>\r\n      <text>A bunch of [[text]] here.</text>\r\n    </revision>\r\n    <revision>\r\n      <text>An earlier [[revision]].</text>\r\n    </revision>\r\n  </page>\r\n</mediawiki>\r\n"

> {:ok, wiki} = SAXMap.from_string(xml)
{:ok, %{"mediawiki" => "\r\n"}}
```

**Comment by xinz** [comment]
Hello, thanks for this reporting, we have fixed it in commit [4eb3302](https://github.com/xinz/sax_map/commit/4eb330243e6da21a306e87bb1dd2f9542fbe0ee0)

**Comment by xinz** [comment]
Released in [1.0.1](https://hex.pm/packages/sax_map/1.0.1)
